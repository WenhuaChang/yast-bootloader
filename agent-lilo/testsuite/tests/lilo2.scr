.

`ag_liloconf(
    LiloConf("lilo", "tests/lilo2.in.test")
)
