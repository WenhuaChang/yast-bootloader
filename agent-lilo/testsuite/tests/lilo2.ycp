/*
 * Authors:
 *  Unknown <unknown@addres.xx>
 *
 * $Id$
 *
 */

{
    // read/write test

    // get first image section name
    string str=select(Dir(.sections), 0, "label");

    list olddir=Dir(.sections);

    // change it's label
//    Write(add(add(.sections, str), "label"), "\"New Label\"");
//    Write(., nil);

    return [olddir, Dir(.sections)];
}
