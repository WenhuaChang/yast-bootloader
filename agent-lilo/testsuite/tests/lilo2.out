([["linux", "suse", "memtest86"], ["linux", "suse", "memtest86"]])
boot	= /dev/hda
vga	= 771
read-only
menu-scheme = Wg:kw:Wg:Wg
prompt
timeout	= 80
message	= /boot/message

  image  = /boot/vmlinuz
  label  = linux
  root   = /dev/hda3

  image  = /boot/vmlinuz.suse
  label  = suse
  root   = /dev/hda3
  optional

  image  = /boot/memtest.bin
  label  = memtest86

